src/wallet/rpcwallet.cpp

```diff
@@ -346,7 +346,7 @@ UniValue listaddresses(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "[\n"
             "  {\n"
-            "    \"source\": \"imported|imported_watchonly|keypool|legacy_seed|mnemonic_seed\"\n"
+            "    \"source\": \"imported|imported_watchonly|keypool|legacy_random|legacy_seed|mnemonic_seed\"\n"
```

Comment:
Which issue will the spurious `keypool` be fixed as part of?

Reply:
Fixed here (in 469137687)